hs628_tools/Visualization.py

```diff
@@ -0,0 +1,294 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import pandas as pd
+import sys 
+import plotly.plotly as py
+import plotly.graph_objs as go
+#plotly.tools.set_credentials_file(username='hhowell', api_key='BRP1aHhfTHwkqnvalVQL')
+from sklearn import datasets, linear_model
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import mean_squared_error,mean_absolute_error, r2_score
+###This is importing a class from Issue #11 Test module. We used the ratio function to determine the test size ratio for the train_test_split function. 
+#from Test import data_split as ds
+
+
+###Bar Plot-Count
+#This function counts all instances in a variable and creates a barplot to display the results 
+#@param: The variable to be counted, the x_axis label, title of the graph
```

Comment:
Although I've seen this kind of @param documentation syntax used with other languages, I'm not familiar with it for python, and PyCharm does not seem to recognize it either.  Best to stick with a syntax that is standard for the language, there is a link or two in one of the class announcements.  And that preceding comment should instead be in a docstring.